platform/darwin/src/MGLSymbolStyleLayer.h

```diff
@@ -1204,6 +1206,15 @@ MGL_EXPORT
  * `MGLCameraStyleFunction` with an interpolation mode of:
    * `MGLInterpolationModeExponential`
    * `MGLInterpolationModeInterval`
+ * `MGLSourceStyleFunction` with an interpolation mode of:
```

Comment:
Also, it is likely that https://github.com/mapbox/mapbox-gl-native/pull/7939 will land before this PR and, I think, this problem will go away then once this branch is rebased on top of that change set.